rpcd/playbooks/roles/rpc_maas/defaults/main.yml

```diff
@@ -464,6 +464,7 @@ maas_pip_packages:
   - rackspace-monitoring-cli
   - ipaddr
   - swift
+  - waxeye
```

Comment:
You're right. I built off the master branch, then ran ``setup-maas.yml`` off this branch here.
Like you said, the waxeye wheel should be built on the repo_server when the repo plays run.